app/views/admin/budgets/_form.html.erb

```diff
@@ -63,6 +63,12 @@
                     class: "button hollow" %>
       <% end %>
 
+      <% if @budget.investments.where(winner: true).count > 0 %>
+        <%= link_to t("budgets.show.see_results"),
+                    budget_results_path(@budget, heading_id: @budget.headings.first),
```

Comment:
This is good too!
Maybe a little more concise like this:
`budget_results_path(@budget)`